test/ziggurat/fixtures.clj

```diff
@@ -99,8 +99,9 @@
                                  :channel-1  #(constantly nil)}}]
     (mount-config)
     (mount-tracer)
+
     (->
-     (mount/only [#'connection])
+     (mount/only [#'connection #'ziggurat.metrics/statsd-reporter])
```

Comment:
Why is `statsd-reporter` needed in this fixture? Also there is a separate mount function `mount-metrics`, why not reuse that?

Reply:
Used statsd-reporter since I wasn't aware of joining fixtures. mount-metrics is a better option.